code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -1903,9 +1904,13 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 		detach()
 		return
 
+/obj/structure/centrifuge/AltClick()

```

Comment:
Personally I greatly dislike these contextual click actions that have no explanation for them anywhere ingame